_posts/2017-09-19-FAF-in-container.markdown

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title:  "FAF in container"
+date:   2017-09-19 10:00:00 +0100
+author: marusak
+visible: 1
+categories: FAF container
+---
+It has never been easier to deploy FAF.
+
+# Why deploying own FAF
+Firstly, standard users probably do not want to deploy own FAF - FAF is here
+to collect a huge amount of crashes from lots of users. Deploying own FAF might
+be interesting for SysAdmins who control hundreds of machines. It also may be
+very interesting on container platform such as OpenShift.
```

Comment:
> very interesting

The intensifier "very" modifies the weak adjective interesting. Consider replacing the phrase with a strong adjective in order to sharpen your writing. E.g.  Fascinating